applications/h264/h264_endoscopy_tool_tracking/python/metadata.json

```
@@ -48,9 +48,19 @@
         }
       ]
     },
-    "run": {
-      "command": "python3 <holohub_app_source>/h264_endoscopy_tool_tracking.py --data <holohub_data_dir>/endoscopy",
-      "workdir": "holohub_bin"
-    }
-  }
+		"modes": {
+			"replayer": {
+				"description": "Use a pre-recorded Endoscopy video as input",
+				"build": {
+					"docker_build_args": ["--target", "base"]
+				},
+				"run": {
+					"command": "python3 <holohub_app_source>/h264_endoscopy_tool_tracking.py --data <holohub_data_dir>/endoscopy",
+					"workdir": "holohub_bin",
+					"docker_build_args": ["--target", "run"]
```

could unset the var using the run args?
```suggestion
					"docker_run_args": ["-e LD_PRELOAD="]
```

---

Gave this a shot but still see the error:
```
[error] [gxf_codelet.cpp:44] Unable to find the GXF type name ('nvidia::gxf::VideoReadBitStream') for the codelet
```

I think `LD_PRELOAD` would need to be set before running `python3` to have effect.

Seems like documenting the user runtime requirement as done in the latest README update is still the best way at the moment:
```
# Note: LD_PRELOAD required to address symbol issue
./holohub run h264_endoscopy_tool_tracking --language python \
    --docker-opts="-e LD_PRELOAD=/opt/nvidia/holoscan/lib/libgxf_core.so"
```